### ISSUE TITLE
operator precedence of AND/OR in where

### ISSUE THREAD

**Issue by kibtidec** [codeschool/sqlite-parser#41]
I see in #20 that operator precedence has been improved/fixed. Nevertheless I still observe an issue with `AND` and `OR`.

The following two expressions result in the same AST, but shouldn't.

```
select x from foo where (a or b) and c
select x from foo where a or b and c
```

Tested with v1.0.0 and also the online demo

**Comment by nwronski** [comment]
You're right, `AND` should compose before `OR` in the second statement. There is an initial fix [in master](https://github.com/codeschool/sqlite-parser/commit/7a51bd6da255fab2408f2f3f891eb74723957a5a) but I will need some more time to test it out before I publish a new release on npm. Thanks!

**Comment by kibtidec** [comment]
Thank you. Fix works for me and doesn't break any of my existing tests. I will add a few more to see if there are any side effects.

**Comment by nwronski** [comment]
Released in v1.0.1, thanks!
